terraform/modules/s3_hosting/main.tf

```diff
@@ -126,6 +126,10 @@ resource "aws_cloudfront_distribution" "client_assets_distribution" {
       ssl_support_method       = "sni-only"
     }
 
+  # Reference the cert validations only so it becomes a dependency
+  tags = {
```

Comment:
nice, didnt know you could do that

Reply:
TF builds a big dependency graph so it can decide what can be run in parallel and what needs to be serial. The problem here though is just that it's too smart. I thought just by providing the validations resource as an argument to this module it would create a dependency, but it looks like we actually have to use it inside the module too.